### PR TITLE
change classes to pten, test=develop

### PR DESCRIPTION
**Pull request by Shixiaowei02**
### PR types

Others
### PR changes

APIs
### Describe

change classes to pten, test=develop

### CODE REVIEW

File: paddle/infrt/dialect/pten/infrt_pten_tensor.td [hunk 1]
```diff
@@ -21,84 +21,36 @@ def PTEN_DenseTensorDialect : Dialect {
...
-      : PDT_Op<"create_inited_tensor." #dtype, [NoSideEffect]> {
-  let summary = "pdt.create_inited_tensor operation";
-
-  let description = [{
-      An operation that creates an tensor with shape and values assigned.
-  }];
-
-  let arguments = (ins I64ArrayAttr:$shape, array_attr:$values);
+class CreateDenseTensorOp<string place, string dtype, string layout> 
+      : PDT_Op<"create_dense_tensor." # place # "." # dtype # "." # layout, [NoSideEffect]> {
```
Comment (Superjomn) [inline paddle/infrt/dialect/pten/infrt_pten_tensor.td:25]: put place, dtype and layout  as argument

File: paddle/pten/backends/cpu/cpu_context.h [hunk 2]
```diff
@@ -27,6 +27,8 @@ namespace pten {
 class CPUContext : public DeviceContext {
  public:
   CPUContext();
+  CPUContext(CPUContext&&);
+  CPUContext& operator=(CPUContext&&);
```
Comment (jiweibo) [inline paddle/pten/backends/cpu/cpu_context.h:31]: 后续其它context是否也需要加move语义？

Comment (Shixiaowei02) [reply paddle/pten/backends/cpu/cpu_context.h:31]: 是的，这是添加到infrt的前提，不影响原有语义

File: paddle/infrt/backends/host/pten_allocator.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```
Comment (jiweibo) [inline paddle/infrt/backends/host/pten_allocator.h:1]: infrt/backends/pten/host
infrt/backends/pten/gpu
infrt/backends/others/host

or

infrt/backends/host/pten
infrt/backends/host/others

**Review by jiweibo — APPROVED** [review]
LGTM

File: paddle/infrt/backends/host/pten_allocator.h (hunk 3 above)
Comment (Shixiaowei02) [reply paddle/infrt/backends/host/pten_allocator.h:1]: > infrt/backends/pten/host infrt/backends/pten/gpu infrt/backends/others/host
> 
> or
> 
> infrt/backends/host/pten infrt/backends/host/others

ok, I will fix in next commit

**Review by Superjomn — APPROVED** [review]
LGTM

File: paddle/infrt/dialect/pten/infrt_pten_tensor.td (hunk 1 above)
Comment (Shixiaowei02) [reply paddle/infrt/dialect/pten/infrt_pten_tensor.td:25]: ok, I will fix in next commit

**Pull request merged**